src/ucp/core/ucp_context.h

```diff
@@ -237,6 +237,12 @@ typedef struct ucp_tl_iface_atomic_flags {
     } atomic32, atomic64;
 } ucp_tl_iface_atomic_flags_t;
 
+/* datatype used to store address bandwidth value.
+   optimization hint: if bandwidth > 0 - it is dedicated value,
+   else - shared value. Assuming that both values can't be not-null */
+typedef struct ucp_tl_bandwidth {
+    float bandwidth;
+} uct_tl_bandwidth_t; /* Interface performance - bandwidth */
```

Comment:
do we need a structure now? Can use just float

Reply:
technically it can be used as single value, but for now it prevents direct access

Reply:
this `bandwidth->bandwidth` looks a bit odd. Maybe just use one float and add many comments?